ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -49,3 +49,11 @@ def eq(input, other, *, out=None):
 
 def argsort(input, dim=-1, descending=False):
     return ivy.argsort(input, axis=dim, descending=descending)
+
+
+def ge(input, other, *, out=None):
```

Comment:
This is me being really picky but could you set greater_equal to be the main function and ge to be the alias. I think its more readable :)

Reply:
Yeah makes sense Let me do it I only implemented this way following torch implementation